README.md

```diff
@@ -104,70 +176,132 @@ Username:
 Password:
 > enter your github token
 
-Click the "*connect*" button at the top of the page to save the connection
+Click the `Connect` button at the top of the page to save the connection
 
-If the information was entered correctly the connection status will be *Successful*
+If configured correctly, the connection status will show as *Successful*
 
 ## Deploying OPAL
-With argoCD running and configured properly, opal applications can be helm installed.
+With argoCD running and configured, the opal helm chart can now be installed.
+
```

Comment:
The confluence page says to close down the port forwarding from above here. Is this necessary?

Reply:
Not usually. Some people have had issues with accessing any of the ingress points if they leave the port-forward open. I've never seen it happen personally